domains/niraj.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NirajShr3stha",
+        "email": "mrshrestha@protonmail.com"
+    },
+    "records": {
+        "A": "129.154.246.82"
```

A records is an instance of array
```suggestion
        "A": ["129.154.246.82"]
```